server/utils/utils.ts

```diff
@@ -18,4 +22,44 @@ const convertToTitleCase = (sentence: string): string =>
 
 const hasRole = (user: Express.User, role: AuthRole): boolean => user?.userRoles.includes(role) || false
 
-export { convertToTitleCase, hasRole }
+/**
+ * Converts a SimpleDateTime display value to a JSON string format dd/mm/yyyy hh:mm
+ * @param dt: SimpleDateTime
+ */
+const simpleDateTimeToJson = (dt: SimpleDateTime): string | undefined => {
+  const { inductionDate, inductionTime } = dt
+  const momentDt = moment(
+    [inductionDate.year, inductionDate.month, inductionDate.day].join('-'),
+    ['YYYY-M-D', 'YY-M-D'],
+    true
+  )
+  if (!momentDt.isValid()) {
+    return undefined
+  }
+  let hour
+  if (inductionTime.ampm === 'pm') {
+    hour = +inductionTime.hour < 12 ? +inductionTime.hour + 12 : +inductionTime.hour
+  } else {
+    hour = +inductionTime.hour === 12 ? +inductionTime.hour - 12 : +inductionTime.hour
+  }
+  momentDt.add(hour, 'hours')
+  momentDt.add(+inductionTime.minute, 'minutes')
+  return momentDt.format('DD/MM/YYYY HH:mm')
+}
+
+/**
+ * Converts a JSON date time `dd/mm/yyyy hh:mm` to a SimpleDateTime for display
+ * @param dt: string
+ */
+const jsonToSimpleDateTime = (dt: string): SimpleDateTime | undefined => {
+  const momentDt = moment(dt, 'D/MM/YYYY HHmm')
```

Comment:
Is this the correct format as expected from the API? No colon?

Reply:
I think the format ignores white-space and non-numeric characters, and the tests (a lot of edge cases tried) are all happy with this, so should be ok?